jdaviz/configs/specviz/plugins/redshift_slider/tests/test_redshift_slider.py

```diff
@@ -0,0 +1,68 @@
+from .. import redshift_slider as rs
+
+import astropy.units as u
+import numpy as np
+import pytest
+from jdaviz.configs.specviz.helper import SpecViz
+from specutils import Spectrum1D
+
+
+SPECTRUM_SIZE = 10  # length of spectrum
+
+
+@pytest.fixture
+def specviz_app():
+    return SpecViz()
+
+
+@pytest.fixture
+def spectrum1d():
+    spec_axis = np.linspace(6000, 8000, SPECTRUM_SIZE) * u.AA
+    flux = (np.random.randn(len(spec_axis.value)) +
```

Comment:
Same as my other review comment before: Avoid use of random or set a seed.

Reply:
Got it, I addressed this in my other PR. I will rebase once that is merged.